sakura_core/uiparts/CGraphics.h

```diff
@@ -262,5 +265,62 @@ class CGraphics{
 	bool				m_bDynamicBrush;	//m_hbrCurrentを動的に作成した場合はtrue
 };
 
+/*!
+ * @brief API関数FillRectの高速版(ブラシ用)
+ *
+ * @param [in] hDC デバイスコンテキスト
+ * @param [in] rc 塗りつぶし対象の矩形
+ * @param [in] hBrush 塗りつぶしに使うブラシハンドル
+ */
+inline bool MyFillRect( const HDC hDC, const RECT &rc, const HBRUSH hBrush ) noexcept
+{
+	assert( hDC );
+	assert( hBrush );
+	assert( !IS_INTRESOURCE( hBrush ) );
+
+	HGDIOBJ hBrushOld = ::SelectObject( hDC, hBrush );
```

Comment:
`SelectObject` の戻り値が `NULL` もしくは `HGDI_ERROR` の時に
```cpp
assert(false);
return false;
```
するのも有りではないかと思いました。まぁしなくても良いと思いますが…。

https://docs.microsoft.com/en-us/windows/desktop/api/wingdi/nf-wingdi-selectobject#return-value

Reply:
HGDI_ERRORが返った場合を処理するようにしました。
NULLが返るのはregionを渡してエラーになったときだけのようなのでそっちはスルーです。

Reply:
> If an error occurs and the selected object is **not** a region, the return value is NULL. Otherwise, it is HGDI_ERROR.

エラーが起きた際に選択された？オブジェクトがリージョンでは無い場合は戻り値が NULL になる、そうでない場合は `HGDI_ERROR` 

と書かれているように見えます。なので NULL チェックした方が良いかなと思いました。自分の解釈が間違ってるかもしれませんしここのエラーチェックを自分でやった記憶が無いので自信無いですが。。あーエラーチェック面倒くさいですね。。